### DISCUSSION TITLE
/usr/local/bin already exists

### DISCUSSION THREAD

**Discussion by jtwohic** [Homebrew/discussions#3863] [Getting started]
When I'm trying to install homebrew.  I get the following message telling me that bin is a file that already exists in the /usr/local/
Deleting it will break a lot of things, any suggestions how I instal homebrew in this scenario?

```
Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /bin/mkdir -p /usr/local/bin
mkdir: /usr/local/bin: File exists
Failed during: /usr/bin/sudo /bin/mkdir -p /usr/local/bin
```

**Comment by SMillerDev** [comment]
It's against the unix file system conventions for that path to be a file so I'm expecting its already breaking everything.

**Comment by jtwohic** [comment]
any suggestions on how to correct this, I didn't define that file myself. 
I have had no issues other than installing homebrew on this MacBook.
